formatting-tools/.clang-tidy

```diff
@@ -41,13 +41,13 @@ CheckOptions:
   - { key: readability-identifier-naming.PublicMethodCase, value: lower_case}
   - { key: readability-identifier-naming.StaticConstantCase, value: lower_case}
   - { key: readability-identifier-naming.StaticVariableCase, value: lower_case}
-  - { key: readability-identifier-naming.StructCase, value: lower_case}
-  - { key: readability-identifier-naming.TemplateParameterCase, value: Camel_Snake_Case}
-  - { key: readability-identifier-naming.TemplateTemplateParameterCase, value: Camel_Snake_Case}
-  - { key: readability-identifier-naming.TypeAliasCase, value: lower_case}
-  - { key: readability-identifier-naming.TypedefCase, value: lower_case}
-  - { key: readability-identifier-naming.TypeTemplateParameterCase, value: Camel_Snake_Case}
-  - { key: readability-identifier-naming.UnionCase, value: lower_case}
+  - { key: readability-identifier-naming.StructCase, value: CamelCase}
+  - { key: readability-identifier-naming.TemplateParameterCase, value: CamelCase}
+  - { key: readability-identifier-naming.TemplateTemplateParameterCase, value: CamelCase}
+  - { key: readability-identifier-naming.TypeAliasCase, value: CamelCase}
+  - { key: readability-identifier-naming.TypedefCase, value: CamelCase}
+  - { key: readability-identifier-naming.TypeTemplateParameterCase, value: CamelCase}
+  - { key: readability-identifier-naming.UnionCase, value: CamelCase}
   - { key: readability-identifier-naming.ValueTemplateParameterCase, value: lower_case}
```

Comment:
`ValueTemplateParameterCase` should be `CamelCase`  #as well.

Reply:
Done.